features/support/rails_template

```diff
@@ -1,3 +1,4 @@
 if Rails.gem_version < Gem::Version.new('6')
   gsub_file "Gemfile", /^gem 'sqlite3'$/, 'gem "sqlite3", "~> 1.3.6"'
+  gsub_file "Gemfile", /^ gem 'spring'$/, ' gem "spring", "!= 2.1.1"'
```

Comment:
I think we want this outside the `if Rails.gem_version < Gem::Version.new('6')` check, since it is failing in every version.

Reply:
Good call! Updated in c301ff9.

I still see one local failing `reloading.feature` test that times out. Is it green for you? Let's see what CI has to say.

Reply:
Although the rails 6 tests passed anyway so 🤷🏻‍♂️

Reply:
Pushed up an additional commit to fix the standard style violation.

Reply:
Thank you! We have a green test suite, should we squash & merge?

Reply:
🎉 Yippee! I'll leave it to you.